build.sh

```diff
@@ -112,7 +112,7 @@ prepare_native_build()
     # Generate version.c if specified, else have an empty one.
     __versionSourceFile=$__scriptpath/bin/obj/version.c
     if [ $__generateversionsource == true ]; then
-        $__scriptpath/Tools/corerun $__scriptpath/Tools/MSBuild.exe "$__scriptpath/build.proj" /t:GenerateVersionSourceFile /p:NativeVersionSourceFile=$__scriptpath/bin/obj/version.c /p:GenerateVersionSourceFile=true /v:minimal
```

Comment:
@maririos may want to chime in on this. 

I,  personally,  like the discoverability (via usage)  when arguments are directly supported in these wrapper scripts, but maintenance and consistency across scripts can be a bit of an annoyance though and problematic if left unsupervised. Ie,  are we supporting an official build switch in every script? What else are we making a part of the script vs some hidden property? 

If we can't be consistent or very intentional about what is supported in these wrapper scripts,  then I'd prefer to leave them extremely basic and provide documentation via some other method (print a link to an MD file with usage perhaps).